Base/home/anon/Documents/sql/select_from_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE SCHEMA TestSchema;
+CREATE TABLE TestSchema.TestTable
+(
+    TextColumn text,
+    IntColumn  integer
+);
+INSERT INTO TestSchema.TestTable (TextColumn, IntColumn)
+VALUES ('Test_1', 42),
+       ('Test_3', 44),
+       ('Test_2', 43),
+       ('Test_4', 45),
+       ('Test_5', 46);
+SELECT *
+FROM TestSchema.TestTable;
```

Comment:
A few files here, like this one, should be changed to end with a newline
![Screenshot from 2021-11-03 09-00-36](https://user-images.githubusercontent.com/5600524/140064496-3d99b693-cadb-48f3-ab65-db458e6fe4fa.png)